src/QuizPage/Answer.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const StyledAnswer = styled.div`
+    background-color: #fff;
+    border: 2px solid ${props => props.borderColor};
+    margin-bottom: 20px;
+    cursor: pointer;
+    font-size: 18px;
+    font-weight: 400;
+    padding: 14px 65px;
+    box-sizing: border-box;
+    flex-basis: 50%;
+`;
+
+const htmlDecode = (input) => {
+    return input.replace(/&#039;/g, "'")
+                .replace(/&quot;/g, "\"");
+}
```

Comment:
since you are using this small util function in two components, do not declare it twice, move it to a separate module, export it, and you can import in both components to use. Currently it's a little WET.